### ISSUE TITLE
ERROR (500) Try To Export Finding Excel

### ISSUE THREAD

**Issue by topahadzi** [DefectDojo/django-DefectDojo#6442]
**Be informative**
ERROR (500) Try To Export Finding Excel

**Bug description**
Got error when try to export finding to excel format

**Steps to reproduce**
1. Go to 'product'
2. Click on 'engagement'
3. Click on 'Findings'
4. Click to export excel
5. See error

**Expected behavior**

**Deployment method** *(select with an `X`)*
- [X] Docker Compose
- [ ] Kubernetes
- [ ] GoDojo

**Environment information**
 - Operating System: [e.g. Ubuntu 18.04]
 - Defectdojo Version v. 2.11.0-dev ( release mode )

**Logs** 
```
[21/Jun/2022 08:23:03] ERROR [django.request:224] Internal Server Error: /reports/excel_export
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/./dojo/reports/views.py", line 939, in excel_export
    worksheet.cell(row=row_num, column=col_num, value=value)
  File "/usr/local/lib/python3.8/site-packages/openpyxl/worksheet/worksheet.py", line 242, in cell
    cell.value = value
  File "/usr/local/lib/python3.8/site-packages/openpyxl/cell/cell.py", line 215, in value
    self._bind_value(value)
  File "/usr/local/lib/python3.8/site-packages/openpyxl/cell/cell.py", line 184, in _bind_value
    raise ValueError("Cannot convert {0!r} to Excel".format(value))
```

**Comment by StefanFl** [comment]
This problem has been fixed with the 2.11.0 release version

**Comment by topahadzi** [comment]
Thanks for the support and after update the image the issue resolve 
You can close this ticket
